thrift/src/main/java/com/intellij/plugins/thrift/annotator/UnresolvedReferenceAnnotator.java

```diff
@@ -0,0 +1,58 @@
+package com.intellij.plugins.thrift.annotator;
+
+import com.intellij.codeInspection.ProblemHighlightType;
+import com.intellij.lang.annotation.AnnotationHolder;
+import com.intellij.lang.annotation.Annotator;
+import com.intellij.lang.annotation.HighlightSeverity;
+import com.intellij.plugins.thrift.lang.psi.ThriftCustomType;
+import com.intellij.plugins.thrift.quickfix.ThriftCreateCustomTypeQuickFix;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiReference;
+import org.jetbrains.annotations.NotNull;
+
+public class UnresolvedReferenceAnnotator implements Annotator {
+  @Override
+  public void annotate(@NotNull PsiElement element, @NotNull AnnotationHolder holder) {
+    // Ensure the Psi Element is a CustomType
+    if (!(element instanceof ThriftCustomType)) {
+      return;
+    }
+
+    ThriftCustomType customType = (ThriftCustomType) element;
+    PsiReference[] references = customType.getReferences();
+
+    if (references.length == 2) {
```

Comment:
In what case there might be two references?

Reply:
inclide.Type